charts/vaultserver/ticket-template.md

```diff
@@ -0,0 +1,121 @@
+This template is designed to contact with AppsCode with troubleshooting and support in general. To troubleshoot and find what to do let's get started with the vaultserver phase.
+
+> Please refer to [KubeVault Docs](https://kubevault.com/docs/latest/guides/) for more about KubeVault.
+
+Let's say you have VaultServer instance in namespace demo.
```

Comment:
Might change it to > Let's say you have VaultServer instance provisioned in a namespace.
Generalize the namespace.